### DISCUSSION TITLE
Not able to delete general keymap for vim.keymap.del("n", "<c-/>")

### DISCUSSION THREAD

**Discussion by moonstruckmoth** [LazyVim/LazyVim#1333] [Q&A] [answered]
I put vim.keymap.del("n", "<c-/>") in lua/config/keymaps.lua but it doesn't work.
I'm able to delete other keymaps, for eg vim.keymap.del("n", "<leader>fT") works fine

TIA

PS: The keymap is not showing up correctly in the saved message, but the title has it

**Comment by dpetka2001** [comment]
```lua
vim.keymap.del({ "n", "t" }, "<c-/>")
```
should do it I believe.

**Reply by moonstruckmoth** [reply]
It deletes the keymap. But I'm facing another problem.

I'm trying to override '\<c-/\>' to require("lazyvim.util")
.float_term() which should open terminal in the root dir.

But instead I still see it opening in current working directory, which is the unoverriden general keymap behavior.

If I use another keymap, say '<c-\\>', behavior is as expected. The same rhs of the keymap opens terminal in root dir.

This is all very strange

**Reply by dpetka2001** [reply]
To open the root_dir the command should be 
```lua
require("lazyvim.util").float_term(nil, {cwd = require("lazyvim.util").get_root()})
```
Without `cwd = require("lazyvim.util").get_root()` it will open in the current working directory.

**Reply by moonstruckmoth** [reply]
I tried passing the cwd options, which I had copied from the **LazyVim/lua/lazyvim/config/keymaps.lua**, but I get the same earlier default behavior.

For me, not passing the cwd option opens terminal in root dir. If I pass the option, it opens in the file folder (equivalent to '%:h')

As I mentioned earlier, if I change keymap key to anything other '\<c-/\>', the keymap gets picked up and I see the correct behavior.

I think for now I'll go with the following config, which is working for me,

![image](https://github.com/LazyVim/LazyVim/assets/3016738/0dddd39a-b85b-47ba-a176-1d71c135b08a)

I'll debug the issue once I have a little more time.

Thanks for your help

**Reply by dpetka2001** [reply, marked as answer]
Strange. For me when I am in `~` directory and open from there a file with `nvim ~/.config/nvim/init.lua`, then:
- `lua require("lazyvim.util").float_term()` opens terminal in `~` directory which is the `cwd`
- `lua require("lazyvim.util").float_term(nil, {cwd=require("lazyvim.util").get_root()})` opens terminal in `~/.config/nvim/` which is the root dir of init.lua file. 

Do you happen to use copilot? There was an issue about root_dir and cwd when using copilot #1293. You might want to keep track of it.

**Reply by moonstruckmoth** [reply]
I tried in LazyVim folders and things are working fine. 
It still doesn't work in my work-related folders, it could be an issue with how it's setup.

BTW, I don't use copilot.

I'll spend sometime to debug my work-related folders later.
For now we can pause this discussion.
